app-connector-client/server/app.js

```diff
@@ -36,7 +35,21 @@ module.exports = function (varkesConfigPath = null, nodePortParam = null) {
     app.get("/", function (req, res) {
         res.render('index', { appName: varkesConfig.name })
     })
-
+    app.get("/logo", function (req, res) {
+        var ext = "image/svg+xml"
+        var img_path = path.resolve(__dirname, 'views/static/logo.svg')
+        if (varkesConfig.logo) {
+            img_path = varkesConfig.logo;
+            var tmp = img_path.split(".")
```

Comment:
The splitting by dot is a bit naive.. what if the file name contains a dot..
That all guessing of content type dependent on image is maybe a bit too much Either we use a library or maybe let's fix the supported types to "svg" only. Then we have no problem with image scaling and we don't have to do magic.

So if a mock wants a custom logo, then it must be provided in svg. Letzt have just a config validation to assure it and we are done.

Reply:
Ok I could limit it to svg only . but the dot will work as long as it's an image file because I am taking the content after the last dot

Reply:
hmm.. that is an argument :)
However, am afraid that other formats will always end up in bad rendering as we do not specify size and are not scaling..

Reply:
I have allowed only svg and added a validation in the config.js file